benchmarking/queries.py

```diff
@@ -79,3 +79,12 @@ def Q2_neo(from_time, to_time):
        '|> mean(column: "_value")')
 
 Q4_neo = "Too slow"
+
+
+"""
+Q6
+"""
+Q6_influx = Q2_influx
+
+Q6_get_timestamps = "" + \
+"MATCH (p {id:$page_id})-[r:LINKED_TO]->() RETURN DISTINCT r.from_timestamp ORDER BY r.from_timestamp ASC;"
```

Comment:
```suggestion
"MATCH (p:Page{id:$page_id})-[r:LINKED_TO]->(Page) RETURN DISTINCT r.from_timestamp ORDER BY r.from_timestamp ASC;"
```
Synes dette er litt tydeligere

Reply:
Enig!